Formalisation(shacl)/Core/PiecesShape/Distribution.ttl

```diff
@@ -21,22 +21,24 @@
   ] ,
   [
     sh:path dct:title ;
-    sh:datatype xsd:string ;
+	sh:nodeKind sh:Literal ;
     sh:minCount 1 ;
     sh:uniqueLang true ;
   ] ,
   [
     sh:path dct:description ;
-    sh:datatype xsd:string ;
+	sh:nodeKind sh:Literal ;
```

Comment:
This will allow for random literals (also numeric typed ones) for a description, I'd highly recommend not to allow that.